watermarker/src/commonTest/kotlin/integrationTest/fileWatermarker/TextWatermarkerIntegrationTest.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2023-2024 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
```

Comment:
If the file itself and the content are created in 2024, the copyright year in the license should be changed to 2024 only.